src/java.base/share/classes/java/util/Spliterator.java

```diff
@@ -295,7 +295,7 @@
  */
 public interface Spliterator<T> {
     /**
-     * If a remaining element exists, performs the given action on it,
+     * If a remaining element exists: performs the given action on it,
```

Comment:
I'm not sure the ":" is an improvement on the ",". 
"If the remaining element exists, the action is performed on it, ..."

Reply:
@RogerRiggs "the" instead of "a" gives me the impression that the Spliterator only has a single element left. The choice of colon instead of a comma is because "If a remaining element exists" introduces a list of 2 elements (and introducing a list is a good place for a colon) whereas the "else returns {@code false}" is a closely connected sentence to the topic of the list, which seems like a place to use the semi-colon.

But, not being a native English-speaker, I will happily change it to something which is more correct. :)

Reply:
The switch from "a" to "the" was unintentional.  
"If a remaining element exists, the action is performed on it, returning true;..."

Reply:
@RogerRiggs Got it. :)